working-together/README.md

```diff
@@ -125,5 +125,6 @@ While we don’t require sharing pronouns, the widespread practice of sharing th
 - Sometimes people’s pronouns change. If a person introduces themself with pronouns that are different from what they used previously, there’s no need to comment on the change.
 - When everyone in a meeting or group shares their pronouns every time, it can implicitly ‘out’ a person who is the only one not sharing their pronouns. If you’re comfortable, you can help by occasionally not sharing your pronouns, especially when the majority of the group is sharing theirs. This can normalize the idea that pronouns are optional and opting out doesn’t imply a specific identity.
 - If you’re asked to share your pronouns and you’d rather not, you do not need to provide an explanation. A simple “I’m not sharing my pronouns today” or “I’m fine with you addressing me by my name.” is enough.
+- If someone has shared their pronouns with you, think carefully before sharing their pronouns in other contexts, especially when such information might be publicly visible. That person may not be comfortable with other people knowing their pronouns, and/or it could put them in danger to be [outed](https://www.hrc.org/resources/glossary-of-terms) (for example, if their housing is dependent on a transphobic person who might evict them upon learning that they're trans).
```

Comment:
Hi I am extremely [nitpicky](https://en.wikipedia.org/wiki/And/or#Criticism) about "and/or". Could we just say "or" instead? I don't think of the English language "or" a strictly "exclusive or" -- you'd need to add "either" to get that.

```suggestion
- If someone has shared their pronouns with you, think carefully before sharing their pronouns in other contexts, especially when such information might be publicly visible. That person may not be comfortable with other people knowing their pronouns, or it could put them in danger to be [outed](https://www.hrc.org/resources/glossary-of-terms) (for example, if their housing is dependent on a transphobic person who might evict them upon learning that they're trans).
```

Reply:
Perhaps `it could also put them in danger...`? In this case I think it passes the definition that 'and/or' is trying to achieve.

Reply:
Can you suggest some more specific guidance around 

>think carefully before sharing their pronouns in other contexts

For example, say Alex has shared with you that their pronouns are "they/them/theirs". 

I think the intent of this change is for people to not do things like send emails saying, "Hey everyone, make sure to use they/them/theirs when talking about Alex."

But what if you're in a client meeting and talking about Alex? Or sharing a story about Alex with a friend? Or talking about Alex on social media ("Alex and I had an awesome lunch together and they really inspired me to try a new menu item")? You have to use their pronouns in these instances so that you're honoring how they want to be referred to.